datasets/proteinfold.yaml

```diff
@@ -0,0 +1,38 @@
+Name: ProteinFold Dataset
+Description: Compilation of genetic and structure databases and model parameters required to run AlphaFold2, ColabFold and ESMFold through the nf-core/proteinfold pipeline. It includes files from popular biological data resources such as UniProt, PDB, MGnify, BFD and other expandable profile databases as well as parameters from pre-trained deep learning models specialized in predicting highly accurate protein structures from protein sequences.
```

Comment:
Maybe include the links to Alphafold2 (https://github.com/deepmind/alphafold), ColabFold (https://github.com/sokrypton/ColabFold) and ESMFold(https://github.com/facebookresearch/esm)?

Reply:
And of course the link to nf-core/proteinfold